packages/integration-karma/test/polyfills/text-content/index.spec.js

```diff
@@ -8,9 +11,36 @@ describe('Node.textContent', () => {
         expect(elm.textContent).toBe('text content');
     });
 
+    it('should not return comment text from 2nd level ELEMENT_NODE', () => {
+        const elm = document.createElement('div');
+        const secondLevelElement = document.createElement('div');
+
+        secondLevelElement.appendChild(document.createComment('2nd level comment'));
+        secondLevelElement.appendChild(document.createTextNode('2nd level text'));
+
+        elm.appendChild(document.createComment('Some comment'));
+        elm.appendChild(document.createTextNode('text content'));
+
+        elm.appendChild(secondLevelElement);
+
+        elm.appendChild(document.createComment('Some other comment'));
```

Comment:
Nit: I think that using `innerHTML` to construct the DOM would make this test easier to understand.